images/fcos-bastion-image/root/usr/bin/hpecmd

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ "${#}" -lt 1 ]; then
+    echo "Usage: ${0} <host_id> <command>"
+    exit 1
+fi
+
+host_id="${1}"
+shift
+
+host_obj=$(sed '1s/^#//; 2,${/^#/d; /^$/d}' "/etc/hosts_pool_inventory" | yq -p csv '.[] | select(.bmc_address == "*'".${host_id}"'*")')
```

Comment:
The second sed command let this script fail for those nodes that are quarantined, but we could relax this constraint as in removing the second command and letting the first one work in the whole file (no 1 before the s)